packages/hw-transport-node-hid/src/TransportNodeHid.js

```diff
@@ -68,6 +69,10 @@ export default class TransportNodeHid extends Transport<string> {
     listenDevicesPollingSkip = conditionToSkip;
   };
 
+  static setListenDevicesDebugMode = (bool: boolean) => {
```

Comment:
it would be better if we would allow to set a log function instead. I moved to this paradigm in ledgerjs recently too, because on Ledger Live we use a custom logger, which is not console.*